packages/eth-format/index.ts

```diff
@@ -2,6 +2,9 @@ const ETH_UNITS = { wei: 0, Gwei: 9, ether: 18 };
 
 export type EthAmount = string | number | bigint;
 export type EtherUnit = keyof typeof ETH_UNITS;
+export type FormatOptions = Partial<{
```

Comment:
Now that we sometimes need `formatWei` and `prettyFormatWei` without a unit, I see why this option was added. However, I think a better approach would be to remove the unit display handling from these functions altogether and keep them focused purely on number formatting. Since we already pass the unit to these functions, we can append it directly where needed, like this:

```typescript
`${formatWei(value, unit) ${unit}`
`${prettyFormatWei(value, unit) ${unit}`
```

By keeping the function inputs minimal, we reduce the risk of a combinatorial explosion from too many options, which makes the function easier to test and maintain overall.

Reply:
The way I see it: formatting involves transforming a value into a more user-friendly and presentable format, which includes displaying the units.

If you just need to convert an amount, you can use `convertWei`.

To reduce test complexity, we can focus on testing each option separately and ensure the output is correct. For example: when testing the unit display option, we only need to verify that it works correctly, regardless of which unit is being targeted.